yapapi/executor/__init__.py

```diff
@@ -337,8 +336,8 @@ async def process_debit_notes() -> None:
                             note_id=debit_note.debit_note_id,
                         )
                     )
-                    allocation = self._get_allocation(debit_note)
                     try:
+                        allocation = self._get_allocation(debit_note)
```

Comment:
I've put `allocation = ...` inside of `try ... except` here, if `debit_note`, which comes from a response to an API call, is invalid then `self._get_allocation()` may throw an exception an we wouldn't like it to crash the whole function (this change is not related to the issue that this PR solves, I've just noticed that there may be a problem and couldn't resist).